cliquet/statsd.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+import statsd
+
+
+def dummy_decorator(f):
+    """Just call the given function."""
+    @wraps(f)
+    def wrapper(*args, **kwargs):
+        return f(*args, **kwargs)
+    return wrapper
+
+
+class StatsdClient(object):
```

Comment:
Well it could be at the module level or at the class level, I prefer to have global variable on a class.